indigo/distribution.yaml

```diff
@@ -10239,8 +10258,6 @@ repositories:
       version: indigo-devel
     release:
       packages:
-      - collada_parser
-      - collada_urdf
```

Comment:
Yes. Should I merge the removal of these two packages from the robot_model repo before this PR is merged? https://github.com/ros/robot_model/pull/197